src/App.spec.ts

```diff
@@ -494,18 +500,11 @@ describe('App', () => {
 
         it('should not exist in the arguments on incoming events that don\'t support say', async () => {
           // Arrange
-          const fakeReceiver = createFakeReceiver();
-          const assertionAggregator = sinon.fake();
-          const dummyAuthorizationResult = { botToken: '', botId: '' };
-          const dummyReceiverEvents = createReceiverEventsWithoutSay('CHANNEL_ID');
-          const overrides = mergeOverrides(
-            withNoopAppMetadata(),
-            withNoopWebClient(),
-            withMemoryStore(sinon.fake()),
-            withConversationContext(sinon.fake.returns(noopMiddleware)),
-          );
```

Comment:
Could we add the comment that points out that `overrides` is prepared in `beforeEach()` here too?